paper-slider.html

```diff
@@ -483,6 +488,10 @@
         'up pageup end': '_incrementKey'
       },
 
+      attached: function() {
+        this._isRTL = window.getComputedStyle(this)['direction'] === 'rtl';
```

Comment:
Yeah, as @keanulee points out `getComputedStyle` will force layout and so not really ideal here. There is [:dir()](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir) CSS pseudo-class  which will solve this problem but it's [not implemented in Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=576815) yet. One workaround is to check for `dir` attribute on the parent and the body.  @sorvell did a quick prototype on this: https://glitch.com/edit/#!/fern-scarf